src/renderer/hooks/useMapServerQuery.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+import ky from 'ky/umd'
+import { useQuery } from '@tanstack/react-query'
+
+// local host and global port number
+// @ts-ignore
+const MAP_SERVER_URL = 'http://127.0.0.1:' + window.mapServerPort
```

Comment:
more of a nit but guessing this is because `window.mapServerPort` is type `any`? maybe use `as` just so that it's clear what type the variable is supposed to be i.e. 

```suggestion
const MAP_SERVER_URL = 'http://127.0.0.1:' + (window.mapServerPort as number)
```

alternatively, could add a runtime check to make sure it's defined as a number and throw an exception if it's not

Reply:
this is a JS file, not TS, so I don't believe this will work

Reply:
ah right sorry, forgot about that 😅 in that case, take it or leave the suggestion

Reply:
I am quite excited to switch our bundler and actually be able to use Ts(x) files....